src/exm-window.c

```diff
@@ -446,7 +462,7 @@ exm_window_init (ExmWindow *self)
     title = IS_DEVEL ? _("Extension Manager (Development)") : _("Extension Manager");
 
     gtk_window_set_title (GTK_WINDOW (self), title);
-    adw_view_switcher_title_set_title (self->title, title);
+    adw_navigation_page_set_title (self->main_view, title);
```

Comment:
From documentation:

> [AdwNavigationPage:title](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/property.NavigationPage.title.html) should always be set to something meaningful, as it will be used as a tooltip for the back button and label in its context menu, as well as read out by the screen reader.

Then I think we should remove this because it will always be the name of the application.

Reply:
Thinking about this again, I think it is better this way, since the title and the tooltip to return cannot have different texts and not setting the title would leave the tooltip empty.